### DISCUSSION TITLE
只缘身在此山中——《开源文化在中国》读后感

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#6] [Announcements]
# 只缘身在此山中——《开源文化在中国》读后感

最近读完了范小青老师的专著《开源文化在中国》，当时在豆瓣上只留下了一句话：“一言难尽，只想到了一个标题，回头写一篇读后感《只缘身在此山中》。”

https://zhuangbiaowei.github.io/2023/01/03/about-open-source-culture.html

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
鸟群效应?
其实,还是样本太少,中国现在每年新增至少42万职业程序员,
而 在中国 这书, 才访谈了 不到100位,值得长期追踪并追加...

GNU 世界没有 Raymond 这类商业操作盘手,导致一直没有对等的案例出现,
可能只有在 RMS 仙去后,才可能发生也又为用变化了...
